utils/getLocalizedLanguageName.ts

```diff
@@ -1,8 +1,53 @@
-export default function getLocalizedLanguageName(key: MaybeRef<string>) {
+import type availableContentLanguages from "./availableContentLanguages";
+
+const englishLanguageNames: {
+  [key in (typeof availableContentLanguages)[number]]: string;
+} = {
+  af: "Afrikaans",
+  bg: "Bulgarian",
+  cs: "Czech",
+  da: "Danish",
+  de: "German",
+  en: "English",
+  el: "Greek",
+  es: "Spanish",
+  et: "Estonian",
+  fr: "French",
+  fi: "Finnish",
+  he: "Hebrew",
+  hr: "Croatian",
+  hu: "Hungarian",
+  it: "Italian",
+  kha: "Khasi",
+  nb: "Norwegian",
+  nl: "Dutch",
+  ml: "Malayalam",
+  pl: "Polish",
+  pt: "Portuguese",
+  ro: "Romanian",
+  ru: "Russian",
+  sl: "Slovenian",
+  ta: "Tamil",
+  tr: "Turkish",
+  zh: "Chinese (Simplified)",
+  yue: "Cantonese",
+};
+
+function getEnglishLanguageName(k: string) {
+  const languageNames: { [key: string]: string } = englishLanguageNames;
+  return languageNames[k];
+}
+
+export default function getLocalizedLanguageName(
+  key: MaybeRef<string>,
+  useNativeName?: boolean,
```

Comment:
I suggest using a boolean parameter. I think that's more readable

Reply:
Isn't this a boolean parameter..?

Reply:
I guess you've already implemented your suggestion...